apps/web/src/store/app/datacenter/index.tsx

```diff
@@ -114,10 +111,8 @@ export function DataCenterPreloader({ children }: React.PropsWithChildren) {
   }, [api]);
   //# endregion
 
-  if (!dataCenter && !dataCenterPromise) {
-    const promise = getDataCenter();
-    api.setState({ dataCenterPromise: promise });
-    promise.then(async dataCenter => {
+  if (!dataCenter) {
```

Comment:
May I ask that `dataCenterPromise` is eagerly fetched on module load in this pr is because `!dataCenter && !dataCenterPromise` could enter multiple times and thus call `getDataCenter` multiple times?
I think it is still possible that `dataCenterPromise.then` call will be called multiple times and thus call `createDefaultWorkspace`, right?

Reply:
See https://github.com/toeverything/AFFiNE/pull/1076